modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -34,27 +38,79 @@ ifdef::google-cloud-platform[]
 
 You can use the `ccoctl gcp create-all` command to automate the creation of GCP resources.
 endif::google-cloud-platform[]
-ifdef::alibabacloud-default,alibabacloud-customizations[]
+ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 [id="cco-ccoctl-creating-at-once_{context}"]
 = Creating credentials for {product-title} components with the ccoctl tool
 
 You can use the {product-title} Cloud Credential Operator (CCO) utility to automate the creation of Alibaba Cloud RAM users and policies for each in-cluster component.
-endif::alibabacloud-default,alibabacloud-customizations[]
+endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
 [NOTE]
 ====
 By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
 ====
 
+== Acquiring the `ccoctl` program
+
+.Procedure
+ 
+. The `ccoctl` is a Linux binary that must run in a Linux environment. Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ~/.pull-secret
```

Comment:
Suggest to insert another two steps before this step, which tell how to get the `$CCO_IMAGE`, thanks! 

FYI
1. Obtain the OpenShift Container Platform release image by running the following command:
......
2. Obtain the CCO container image from the OpenShift Container Platform release image by running the following command:
......
3. Extract the `ccoctl` binary from the CCO container image within the OpenShift Container Platform release image by running the following command:
......
4. Change the permissions to make `ccoctl` executable by running the following command:
......

Reply:
This is wonderful, @jianli-wei and I very much appreciate your guidance on this! I have added these steps and used the information you directed me to. PTAL!